### PR TITLE
85655 code uplifting automation

### PR DESCRIPTION
**Pull request by mayur1501364**
- Code Refactoring
- Removed unused namespaces
- Removed unused variables and methods
- Removed unused configurations from appsettings
- Renamed variable names and method

### CODE REVIEW

**Review by RaviK2233 — APPROVED** [review]
Reviewed FT and looks good

**Review by Sadha1501493 — CHANGES_REQUESTED** [review]

**Review by jrippington — APPROVED** [review]
Looks good

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs [hunk 1]
```diff
@@ -785,14 +683,14 @@ public static bool verifyOrderOfActions(JsonPayloadHelper jsonPayload, string ge
...
         }
 
-        public static bool verifyInitialXMLHeaders(JsonPayloadHelper jsonPayload, string generatedXMLFilePath)
+        public static bool VerifyInitialXMLHeaders(JsonPayloadHelper jsonPayload, string generatedXMLFilePath)
         {
-            bool b = verifyNOOFACTIONSHeader(jsonPayload, generatedXMLFilePath);
-            bool a = verifyRECTIMEHeader(jsonPayload, generatedXMLFilePath);
-            bool c = verifyCORRIDHeader(jsonPayload, generatedXMLFilePath);
-            bool d = verifyORGHeader(jsonPayload, generatedXMLFilePath);
+            bool isNoOfFactionsMatching = VerifyNOOFACTIONSHeader(jsonPayload, generatedXMLFilePath);
```
[the remark this answers is not in the document]
Comment (mayur1501364) [reply tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs]: Renamed variable and method name

**Review by Vishal14583 — APPROVED** [review]
LGTM

**Review by Sadha1501493 — APPROVED** [review]
LGTM

**Pull request merged**